common/src/main/java/org/dash/wallet/common/ui/dialogs/OffsetDialogFragment.kt

```diff
@@ -46,6 +46,10 @@ open class OffsetDialogFragment : BottomSheetDialogFragment() {
         super.onViewCreated(view, savedInstanceState)
 
         dialog?.setOnShowListener { dialog ->
+            if (!this@OffsetDialogFragment.isAdded) {
+                return@setOnShowListener
+            }
```

Comment:
My guess is that this listener is sometimes called after the dialog is dismissed. Some simulations with a delay support this theory.